packages/components/psammead-brand/src/index.jsx

```diff
@@ -236,14 +226,9 @@ const Brand = props => {
       scriptLink={scriptLink}
       {...rest}
     >
-      <SvgWrapper>
+      <SvgWrapper ref={ref} tabIndex="-1">
```

Comment:
I can see why we've added `tabIndex="-1"` here, but I am not sure we would ever want to enable the calling of focus on a div unless there was a particular requirement to do so. The children of this element are focussable by default, so unless this has been added for another reason, we should probably keep it as that.

Reply:
Initially I thought we could put focus on the SVG but there is a `focusable="false"` attribute on it, I think it's still possible to put focus on it and having just looked into it the `focusable="false"` stops the svg being keyboard focussable, which it is by default. The other option is putting focus on the `LocalisedBrandName` which returns a `VisuallyHiddenText`, so a `span`. In which case, a `tabindex="-1"` would have to be added to the `VisuallyHiddenText`, since I don't think .focus() can be called on a span, or at least it didn't seem to work when I tried it.